### PR TITLE
feat: implement responsiveness for contributors page

### PR DESCRIPTION
**Pull request by karthiknadar1204**

### CODE REVIEW

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
Not looking good for iPad devices and large screens
image


Also, for normal laptop devices, the buttons don't have margins around it and the color on hover is also not looking good. It also seems like there's no hierarchy in font sizes which results in bad UX. Fix it

**Comment by karthiknadar1204** [comment]
will do it right away @Susmita-Dey

**Comment by Susmita-Dey** [comment]
I've having exams so won't be active to help you out or merge PR, if you need help them join our Discord server. Link in the website. Also, @devs-in-tech/devsintech-web-team kindly check it

**Comment by karthiknadar1204** [comment]
@Susmita-Dey Done with the changes for the larger screen sizes,pushing the changes in a bit

**Comment by karthiknadar1204** [comment]
Please check whenever free

File: src/pages/Contributors.js [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+//           </div>
+//         ))}
+//       </div>
+//       <Footer />
+//     </>
+//   );
+// };
+
+// export default Contributors;
+
```
Comment (Susmita-Dey) [inline src/pages/Contributors.js:76]: Can you remove this duplicate lines which are commented?

The file would look clean

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]

**Comment by karthiknadar1204** [comment]
yes, was testing the new code so had commented out, will do and push changes in a bit

**Comment by karthiknadar1204** [comment]
Done, could you please check now

**Review by Susmita-Dey — APPROVED** [review]
LGTM. Thanks for contributing:)

**Pull request merged**